common/src/main/java/org/ergoplatform/appkit/Address.java

```diff
@@ -279,7 +278,7 @@ public String toString() {
 
     @Override
     public int hashCode() {
-        return Objects.hashCode(_address.hashCode(), _address.networkPrefix());
+        return Objects.hash(_address.hashCode(), _address.networkPrefix());
```

Comment:
What is the motivation for this change?

Reply:
com.google.common.base.Objects.hashCode is a method in the Guava library.
java.util.Objects.hash is a method in the JDK.

Their implementation is the same. The motivation is that I removed Guava.